### PR TITLE
fix(condo): DOMA-9843 fix marketSetting formValuePreprocessor

### PR DESCRIPTION
**Pull request by VKislov**
image
"Variable \"$data\" got invalid value { dv: 1, sender: { dv: 1, fingerprint: \"ZOEdK5UUn9Ru\" }, organization: { connect: [Object] }, isPaymentCashTypeBlocked: false, residentAllowedPaymentTypes: [\"online\", \"cash\"] }; Field \"isPaymentCashTypeBlocked\" is not defined by type \"MarketSettingCreateInput\"."

### CODE REVIEW

File: apps/condo/domains/marketplace/components/MarketSetting/MarketSettingForm.tsx [hunk 1]
```diff
@@ -74,6 +74,7 @@ export const MarketSettingForm: React.FC<IMarketSettingForm> = ({ marketSetting,
                 const residentAllowedPaymentTypes = [INVOICE_PAYMENT_TYPE_ONLINE]
                 if (!values[IS_CASH_PAYMENT_TYPE_BLOCKED]) residentAllowedPaymentTypes.push(INVOICE_PAYMENT_TYPE_CASH)
                 values.residentAllowedPaymentTypes = uniq(residentAllowedPaymentTypes)
+                values.isPaymentCashTypeBlocked = undefined
```
Comment (toplenboren) [inline apps/condo/domains/marketplace/components/MarketSetting/MarketSettingForm.tsx:77]: ?

**Review by SavelevMatthew — CHANGES_REQUESTED** [review]
Why do you pass field that not even exist?
And why do you mask it undefined?

File: apps/condo/domains/marketplace/components/MarketSetting/MarketSettingForm.tsx (hunk 1 above)
Comment (Alllex202) [inline apps/condo/domains/marketplace/components/MarketSetting/MarketSettingForm.tsx:77]:
```suggestion
                values[IS_CASH_PAYMENT_TYPE_BLOCKED]= undefined
```

**Review by Alllex202 — APPROVED** [review]

**Comment by VKislov** [comment]
> Why do you pass field that not even exist? And why do you mask it undefined?

Because it comes from Form.Item, but it is not represented in the mutation input, so I overwrite it in the mutation preprocessor

**Review by SavelevMatthew — APPROVED** [review]

**Pull request merged**